ERROR:django.request:Internal Server Error: /api/testcase/editor/Ping Host
Traceback (most recent call last):
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/utils/deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via PUT, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining PUT data. Change your form to point to 172.16.0.197:8000/api/testcase/editor/Ping%20Host/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
ERROR:django.request:Internal Server Error: /api/testcase/editor/Ping Host
Traceback (most recent call last):
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/utils/deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/vi/.venv/lib/python3.11/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via PUT, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining PUT data. Change your form to point to 172.16.0.197:8000/api/testcase/editor/Ping%20Host/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
